### PR TITLE
Add example of natsort usage in documentation

### PR DESCRIPTION
**Pull request by D-Bhatta**
Illustrate briefly in the documentation how to use natsort to sort files
when loading images.

In case of sorting OS generated file paths, sorted() would not be
sufficient:
>>> list_files = os.listdir('.')
>>> list_files
['01.png', '010.png', '0101.png', '0190.png', '02.png']
>>> sorted(list_files)
['01.png', '010.png', '0101.png', '0190.png', '02.png']

In cases like these the ns.PATH algorithm along with natsorted() would be
better:
>>> list_files = natsorted(list_files, alg=ns.PATH)
>>> list_files
['01.png', '02.png', '010.png', '0101.png', '0190.png']

## Description

Closes https://github.com/scikit-image/scikit-image/issues/4513 (added by @soupault ).

## Checklist

- [ ] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- [ ] Gallery example in `./doc/examples` (new features only)
- [ ] Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- [ ] Unit tests
- [ ] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)

## For reviewers

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.

### CODE REVIEW

File: doc/source/user_guide/getting_started.rst [hunk 1]
```diff
@@ -41,3 +41,18 @@ from image files, using :func:`skimage.io.imread`: ::
     >>> from skimage import io
     >>> moon = io.imread(filename)
 
+Use natsort to load multiple images: ::
```
Comment (soupault) [inline doc/source/user_guide/getting_started.rst:44]:
```suggestion
Use `natsort <https://pypi.org/project/natsort/>`_ to load multiple images ::
```

File: doc/source/user_guide/getting_started.rst [hunk 2]
```diff
@@ -41,3 +41,18 @@ from image files, using :func:`skimage.io.imread`: ::
...
+
+    >>> import os
+    >>> from natsort import natsorted, ns
+    >>> from skimage import io
+    >>> list_files = os.listdir('.')
+    >>> list_files
+    ['01.png', '010.png', '0101.png', '0190.png', '02.png']
+    >>> list_files = natsorted(list_files, alg=ns.PATH)
```
Comment (soupault) [inline doc/source/user_guide/getting_started.rst:52]: `alg=ns.PATH` seems to be redundant:
```
In [4]: list_files = ['01.png', '010.png', '0101.png', '0190.png', '02.png']
In [5]: list_files = natsort.natsorted(list_files)                              

In [6]: list_files                                                              
Out[6]: ['01.png', '02.png', '010.png', '0101.png', '0190.png']
```

File: doc/source/user_guide/getting_started.rst (hunk 1 above)
Comment (D-Bhatta) [reply doc/source/user_guide/getting_started.rst:44]: @soupault Thanks for input. I didn't know to do this, so this is a big help. Thank you, and I'll be commit-ing your change.

File: doc/source/user_guide/getting_started.rst (hunk 2 above)
Comment (D-Bhatta) [reply doc/source/user_guide/getting_started.rst:52]: I agree with this. ns.PATH _does_ seem to be suitable for edge cases as defined [here](https://natsort.readthedocs.io/en/master/examples.html#path-sort).

For most use cases, ns.PATH is redundant.

File: doc/source/user_guide/getting_started.rst (hunk 1 above)
Comment (soupault) [reply doc/source/user_guide/getting_started.rst]: @D-Bhatta I have looked it up myself in the same file :sweat_smile:. It is difficult to remember many things at once.

**Comment by soupault** [comment]
@D-Bhatta thanks again and looking forward to seeing further PRs from you :).

**Pull request merged**